Jenkinsfile

```diff
@@ -63,8 +63,11 @@ for (int i = 0; i < splits.size(); i++) {
          retry(count: 2, conditions: [agent(), nonresumable()]) {
           node('docker-highmem') {
             checkout scm
+            def browser = 'firefox'
             def image = docker.build('jenkins/ath', '--build-arg uid="$(id -u)" --build-arg gid="$(id -g)" ./src/main/resources/ath-container/')
-            image.inside('-v /var/run/docker.sock:/var/run/docker.sock --shm-size 2g') {
+            sh 'mkdir -p target/ath-reports && chmod a+rwx target/ath-reports'
```

Comment:
This change and the lines below are to create a directory on the Docker host for ATH reports so that we can later send them to Launchable. Why not install Launchable inside the ATH container and send the data to Launchable from inside the container, you might ask. Well, that would work as well, but this method is consistent with how test result collection is taking place in Jenkins core ATH builds, and it also enables us to use the common Launchable infrastructure used in other places like core and BOM. So, for example, when https://github.com/jenkins-infra/helpdesk/issues/3484 is implemented, we would get that for free here rather than having to maintain a parallel implementation of Launchable package installation in this repository.